database/connector.py

```diff
@@ -73,3 +73,25 @@ def insert_other_network_event(ip, port, machine_name):
             "country": str(IP2Location.get_country_short(ip).decode())
         }
     ).inserted_id
+
+
+def insert_honeypot_events_from_module(ip,username,password,module_name):
```

Comment:
maybe can change the name to `insert_honeypot_events_from_module_processor`, and add the time in args, we want the time of event not the time of submitting the event into db.